kubernetes/resource_kubernetes_config_map_v1_data_test.go

```diff
@@ -88,6 +90,53 @@ func TestAccKubernetesConfigMapV1Data_validation(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesConfigMapV1Data_binaryData(t *testing.T) {
+	name := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	resourceName := "kubernetes_config_map_v1_data.test"
+	baseDir := "."
+	cwd, _ := os.Getwd()
+	if filepath.Base(cwd) != "kubernetes" { // running from test binary
+		baseDir = "kubernetes"
+	}
```

Comment:
What's this for?

Reply:
When creating the test, I gained inspiration from TestAccKubernetesConfigMapV1_binaryData
`func TestAccKubernetesConfigMapV1_binaryData(t *testing.T) {
	var conf corev1.ConfigMap
	prefix := "tf-acc-test-gen-"
	resourceName := "kubernetes_config_map_v1.test"
	baseDir := "."
	cwd, _ := os.Getwd()
	if filepath.Base(cwd) != "kubernetes" { // running from test binary
		baseDir = "kubernetes"
	}

	resource.ParallelTest(t, resource.TestCase{
		PreCheck:          func() { testAccPreCheck(t) },
		IDRefreshName:     resourceName,
		IDRefreshIgnore:   []string{"metadata.0.resource_version"},
		ProviderFactories: testAccProviderFactories,
		CheckDestroy:      testAccCheckKubernetesConfigMapV1Destroy,
		Steps: []resource.TestStep{
			{
				Config: testAccKubernetesConfigMapV1Config_binaryData(prefix, baseDir),
				Check: resource.ComposeAggregateTestCheckFunc(
					testAccCheckKubernetesConfigMapV1Exists(resourceName, &conf),
					resource.TestCheckResourceAttr(resourceName, "binary_data.%", "1"),
					resource.TestCheckResourceAttr(resourceName, "data.%", "1"),
					resource.TestCheckResourceAttr(resourceName, "data.two", "second"),
				),
			},
			{
				Config: testAccKubernetesConfigMapV1Config_binaryData2(prefix, baseDir),
				Check: resource.ComposeAggregateTestCheckFunc(
					testAccCheckKubernetesConfigMapV1Exists(resourceName, &conf),
					resource.TestCheckResourceAttr(resourceName, "binary_data.%", "3"),
					resource.TestCheckResourceAttr(resourceName, "binary_data.raw", "UmF3IGRhdGEgc2hvdWxkIGNvbWUgYmFjayBhcyBpcyBpbiB0aGUgcG9k"),
					resource.TestCheckResourceAttr(resourceName, "data.%", "1"),
					resource.TestCheckResourceAttr(resourceName, "data.three", "third"),
				),
			},
		},
	})
}`